packages/web/src/components/organisms/PropertyCardList/PropertyCard.tsx

```diff
@@ -10,18 +12,25 @@ interface Props {
 
 export const PropertyCard = ({ propertyAddress }: Props) => {
   const { totalRewardsAmount } = useGetTotalRewardsAmount(propertyAddress)
+  const { data } = useGetPropertyAuthenticationQuery({ variables: { propertyAddress } })
+  const includeAssets = useMemo(
+    () => data && truncate(data.property_authentication.map(e => e.authentication_id).join(', '), 17),
+    [data]
+  )
+  const { assetStrength: maybeAssetStrength } = useAssetStrength(
+    data?.property_authentication[0]?.metrics,
+    data?.property_authentication[0]?.market
+  )
+  const assetStrength = useMemo(() => maybeAssetStrength || 0, [maybeAssetStrength])
```

Comment:
I don't want to fall back in the component, so I use use useMemo to do the fallback.